config/initializers/00_devise.rb

```diff
@@ -10,8 +10,8 @@
   # Configure the e-mail address which will be shown in Devise::Mailer,
   # note that it will be overwritten if you use your own mailer class
   # with default "from" parameter.
-  config.mailer_sender = if AppConfig.table_exists? && !AppConfig.first.nil?
-                           AppConfig.first.admin_email
+  config.mailer_sender = if AppConfig.table_exists?
```

Comment:
This is what's breaking the tests (password resets come from Devise) - not sure why the logic is failing but we might have to test `if AppConfig.table_exists? && AppConfig.check :admin_email` instead.